roles/set_openstack_containers/tasks/main.yml

```diff
@@ -105,14 +109,17 @@
   ansible.builtin.include_role:
     name: 'install_yamls_makes'
     tasks_from: 'make_wait'
+  when:
+    - operator_name != 'openstack'
+    - not cifmw_architecture_scenario is defined
```

Comment:
Oki, if we want to run this all the time where 

```
  NOT  (operator_name == 'openstack' AND cifmw_architecture_scenario is defined)
```

That should be
```
  operator_name != 'openstack' OR not cifmw_architecture_scenario is defined
```
so if cifmw_arch_scenario is defined but operator name is not openstack this can still run ... but we don't want to run install_yaml when cifmw_architecture_scenario is defined.

I'm not sure now if we need the operator_name conditional.

Reply:
Strictly speaking this code path has never been used, but I'm leaving it just in case there will be use case for modifying envs in operator other than openstack-operator.
We don't want to run it in cifmw_architecture_scenario and operator_name != 'openstack'. We want to run it only in the context of install_yamls jobs and only  operator_name != 'openstack' for instace baremetal-operator.